src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2289,6 +2327,81 @@ components:
             example: 
             - "path/to/file1"
             - "path/to/file2"
+    JobQueue:
```

Comment:
Please create model classes for `JobQueue` and `ShowJob` in `src/www/ui/api/Models/`

Reply:
Sure, I'll do that!